.pre-commit-config.yaml

```
@@ -0,0 +1,13 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: check-yaml
+      - id: check-json
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: requirements-txt-fixer
+  - repo: https://github.com/psf/black
+    rev: 23.11.0
+    hooks:
+      - id: black
```

```suggestion
      - id: black
        args:  ["--line-length", "120"]
```
We use black formatter with this additional arg.

---

Done